cmake/kokkoskernels_tpls.cmake

```diff
@@ -408,9 +408,7 @@ IF (NOT KOKKOSKERNELS_HAS_TRILINOS)
   KOKKOSKERNELS_IMPORT_TPL(BLAS INTERFACE)
   KOKKOSKERNELS_IMPORT_TPL(LAPACK INTERFACE)
   KOKKOSKERNELS_IMPORT_TPL(MKL INTERFACE)
-  KOKKOSKERNELS_IMPORT_TPL(CUBLAS)
-  KOKKOSKERNELS_IMPORT_TPL(CUSPARSE)
-  KOKKOSKERNELS_IMPORT_TPL(CUBLAS)
+  KOKKOSKERNELS_IMPORT_TPL(CUBLAS INTERFACE)
   KOKKOSKERNELS_IMPORT_TPL(CUSPARSE)
```

Comment:
Why doesn't CUSPARSE also have INTERFACE target property?

Reply:
It has to do with whether it's a single library or involves multiple libraries. You can only have a single `IMPORTED_LOCATION` for an imported library.